client/src/views/User.vue

```diff
@@ -1,82 +1,73 @@
 <template>
-  <div class="user-wrapper">
-    <div class="user-header">
-      <div
-        class="user-header__photo async async--img"
-        :style="{ backgroundImage: `url(${user.photo})` }"
-      ></div>
-      <h1 class="user-header__title async async--text">{{ user.name }}</h1>
-      <p class="user-header__email async async--text">{{ user.email }}</p>
-    </div>
-
-    <div :key="user.lvl" class="user-xp">
-      <i
-        class="user-xp__level fas fa-star"
-        :style="{ color: hashColor(user.lvl) }"
-      >
-        <span class="user-xp__text">
-          {{ user.lvl != null ? user.lvl : "?" }}
-        </span>
-      </i>
-      <div class="user-xp__bar">
-        <span class="user-xp__text" :style="xpTextStyle(user.lvl)">
-          {{ user.lvlXp != null ? user.lvlXp : "-" }} /
-          {{ user.reqLvlXp != null ? user.reqLvlXp : "-" }} xp
-        </span>
+  <div ref="userWrapper" class="user-wrapper">
+    <div ref="userProfile">
```

Comment:
```suggestion
    <div>
```
If `ref="userProfile"` is not used anymore, we can remove it